### PR TITLE
chore: delete WalletConnect v1

### PR DESCRIPTION
**Pull request by vm**
Deletes all WalletConnect v1 related code now that the feature flag is removed.

### CODE REVIEW

File: src/state/user/reducer.ts [hunk 1]
```diff
@@ -125,9 +125,16 @@ const userSlice = createSlice({
     // for all existing users with a previous version of the state in their localStorage.
     // In order to avoid this, we need to set a default value for each new property manually during hydration.
     builder.addCase(updateVersion, (state) => {
-      // If `selectedWallet` is ConnectionType.UNI_WALLET (deprecated) switch it to ConnectionType.UNISWAP_WALLET
-      if (state.selectedWallet === 'UNIWALLET') {
-        state.selectedWallet = ConnectionType.UNISWAP_WALLET
+      // If `selectedWallet` is a WalletConnect v1 wallet, reset to default.
+      if (state.selectedWallet) {
```
Comment (zzmp) [inline src/state/user/reducer.ts:129]: @just-toby does this need a migration?

**Review by zzmp — APPROVED** [review]

File: src/state/user/reducer.ts (hunk 1 above)
Comment (just-toby) [reply src/state/user/reducer.ts:129]: yes i think this makes sense

File: src/components/Identicon/__snapshots__/StatusIcon.test.tsx.snap [hunk 2]
```diff
@@ -112,9 +112,9 @@ exports[`StatusIcon with account renders children in correct order 1`] = `
     class="c1"
   >
     <img
-      alt="Install MetaMask icon"
```
Comment (vm) [inline src/components/Identicon/__snapshots__/StatusIcon.test.tsx.snap:115]: this seems wrong

**Review by zzmp — APPROVED** [review]

File: src/components/Identicon/__snapshots__/StatusIcon.test.tsx.snap (hunk 2 above)
Comment (zzmp) [reply src/components/Identicon/__snapshots__/StatusIcon.test.tsx.snap:115]: tested manually and it looks to be working correctly.

Comment (vm) [reply src/components/Identicon/__snapshots__/StatusIcon.test.tsx.snap:115]: zach tested and says it's okay

**Review by just-toby — APPROVED** [review]

**Pull request merged**